content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md

```diff
@@ -0,0 +1,157 @@
+---
+weight: 40
+title: Release 10.16.0.0
+layout: bundle
+---
+
+### Improvements
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>DTM application is integrated with Gainsight to capture the data.</td>
+<td>CTM-130</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>Graceful handling of custom property and asset type deletion from backend is implemented.</td>
+<td>CTM-141</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>On delete operation in Property library page confirmation pop-up is shown.</td>
+<td>CTM-159</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>Implemeted maximum file size field for custom property of file type.</td>
+<td>CTM-175</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>Graceful handling of asset type edit case when it has instances created out of it.</td>
+<td>CTM-144</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>Graceful handling of custom property edit case when existing asset types/assets are using it.</td>
+<td>CTM-147</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>Display of Alert save pop-up in Localization page.</td>
+<td>CTM-155</td>
+</tr>
+
+</tbody></table></div>
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>Import Assets button is not visible for last asset in the hierarchy now.</td>
```

Comment:
```suggestion
<td>The Import Assets button is not visible for the last asset in the hierarchy now.</td>
```
this sounds a bit like it describes the situation prior to fixing - should it really be invisible?

Reply:
```suggestion
<td>Th **Import assets** button is no longer visible for last asset in the hierarchy.</td>
```